proto/Dockerfile

```diff
@@ -33,3 +36,4 @@ ENV XDG_CACHE_HOME /tmp/.cache
 RUN chmod -R 775 /usr/bin/
 RUN chmod -R 775 /usr/include/google
 RUN chmod -R 775 /go
+USER 65532:65532
```

Comment:
Would you mind explaining the reason behind this change? Thanks!

Reply:
I needed temporary root privilege  to install packages. I saw this usage pattern [here](https://github.com/ray-project/kuberay/blob/9c28b7de1eefc834d779f35acf5927e8d3c413c4/ray-operator/Dockerfile#L18-L24). @anishasthana Could you take a look if `USER 65532:65532` is necessary ?

Reply:
Thanks!